### PR TITLE
Adding Media visualization

### PR DESCRIPTION
**Pull request by colinbendell**

### CODE REVIEW

**Comment by bazzadp** [comment]
Let me know when you're done here @colinbendell and I'll review, merge and then open a new branch for copyediting.

File: src/content/en/2019/media.md [hunk 1]
```diff
@@ -76,14 +140,31 @@ Each format has its own merits and has ideal uses for the web. A very simplified
...
-
-
 ## Image Formats
+
 In aggregate, across all page, we indeed see the prevalence of these formats. JPEG, one of the oldest formats on the web, is by far the most commonly used image formats at 60% of the image requests and 65% of all image bytes. Interestingly, PNG is the second most commonly used image format 28% of image requests and bytes. The ubiquity of support along with the precision of color and creative content are likely explanations for its wide use. In contrast SVG, GIF and WebP share nearly the same usage at 4%.
-(figure image formats by % of total requests)
+
+<figure>
+  <a href="/static/images/2019/04_Media/image_format_usage.png">
+    <img alt="Most frequent Image types used on webapges" src="/static/images/2019/17_CDN/html_cdn_usage.png" width="600">
```
Comment (rviscomi) [inline src/content/en/2019/media.md]:
```suggestion
    <img alt="Most frequent Image types used on web pages" src="/static/images/2019/04_Media/image_format_usage.png" width="600">
```

File: src/content/en/2019/media.md [hunk 2]
```diff
@@ -76,14 +140,31 @@ Each format has its own merits and has ideal uses for the web. A very simplified
...
+|png|28.2|
+|webp|4.2|
+|gif|3.67|
+|svg|3.63|
+
+
 Of course, web pages are not uniform in their use of image content. Some depend on images more than others. Look no further than the home page of Google.com and you will see very little imagery compared to a typical news website. Indeed, the median website has 13 images and 61 at the 90th percentile and a whopping 229 at the 99th percentile.
 (figure image frequency per page)
 
+||||||||
```
Comment (rviscomi) [inline src/content/en/2019/media.md]: Is this needed?

File: src/content/en/2019/media.md [hunk 3]
```diff
@@ -76,14 +140,31 @@ Each format has its own merits and has ideal uses for the web. A very simplified
...
-(figure image formats by % of total requests)
+
+<figure>
+  <a href="/static/images/2019/04_Media/image_format_usage.png">
+    <img alt="Most frequent Image types used on webapges" src="/static/images/2019/17_CDN/html_cdn_usage.png" width="600">
+  </a>
+  <figcaption>Figure 6: Image Format Usage</figcaption>
+</figure>
+
+* Image Format Usage
```
Comment (rviscomi) [inline src/content/en/2019/media.md]: Tables should also be wrapped in figures, eg

```<figure markdown>...```

File: src/content/en/2019/media.md [hunk 4]
```diff
@@ -15,14 +15,30 @@ last_updated: 2019-11-07T21:46:11.000Z
...
-(% of image bytes per page figure)
 
-While media is critical for the visual experience, the impact of this high volume of bytes has two side effects. First, the network overhead required to download these bytes can be large and in cellular or slow network environments (like coffee shops or tethering when in an uber) can dramatically slow down the page performance. Images are a lower priority request by the browser, but can easily block CSS and JavaScript in the download. This by itself can delay the page rendering. Yet at other times, the image content is the visual queue to the user that the page is ready and slow transfers can give a perceived slowness of the page.
+<figure>
+  <iframe aria-labelledby="fig1-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1189524305&format=interactive"></iframe>
+  <a href="/static/images/2019/04_Media/fig1.png" class="fig-mobile">
+    <img src="/static/images/2019/04_Media/fig1.png" aria-labelledby="fig1-caption" width="600">
```
Comment (rviscomi) [inline src/content/en/2019/media.md]: Do you need help generating figN.png images?

File: src/content/en/2019/media.md [hunk 5]
```diff
@@ -129,93 +243,207 @@ While previously it appeared that GIF files were smaller than JPEG, we can now c
...
+## Video
 While images dominate the media being served on web pages, videos are beginning to have a major role in content delivery on the web.  According to HTTP Archive, we find that 4.06% of desktop and 2.99% of mobile sites are self hosting video files (meaning that the files are not hosted at websites like YouTube or Facebook.).
 
+### Video Formats
+
 Video can be delivered with many different formats, and players.  The dominant formats for mobile and desktop are .ts (segments of HLS streaming) and .mp4 (the H264 MPEG):
- (figure (count of video files by extension)
+
+<figure>
+  <iframe aria-labelledby="fig19-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://todo"></iframe>
```
Comment (rviscomi) [inline src/content/en/2019/media.md]: These `todo`s need to be resolved before merging. They're causing the iframes to fail.

image

File: src/content/en/2019/media.md [hunk 6]
```diff
@@ -76,146 +143,315 @@ Each format has its own merits and has ideal uses for the web. A very simplified
...
 
 While other format features simply organize the content and sometimes requires contextual knowledge. For example, applying progressive encoding of a JPEG reorganizes the pixels into scan layers that allows the browser to complete layout sooner and coincidently reduces pixel volume.
 
 One Lighthouse test is an A/B comparing baseline with a progressively encoded JPEG. This provides a smell to indicate whether the images overall can be further optimized with lossless techniques and potentially with lossy techniques like using different quality levels.  
- (figure % optimized images)
+
+ <figure>
+  <iframe aria-labelledby="fig11-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://todo"></iframe>
```
Comment (rviscomi) [inline src/content/en/2019/media.md]: Replace todo URLs

**Pull request merged**